core/src/main/java/nl/nn/adapterframework/doc/model/FrankDocGroup.java

```diff
@@ -0,0 +1,20 @@
+package nl.nn.adapterframework.doc.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.Getter;
+
+public class FrankDocGroup {
+	private @Getter String name;
+	private @Getter Map<String, FrankElement> elements;
+
+	public FrankDocGroup(String name) {
+		this.name = name;
+		this.elements = new HashMap<>();
+	}
+
+	public void addAsElement(Class<?> clazz, FrankDocModel model) {
```

Comment:
Ik zie deze method nog niet gebruikt worden buiten de tests. Is er logica of configuratie die bepaalt welke element er in een groep zitten?

Reply:
Nog niet. De logica die bepaalt welke groepen en welke elementen er zijn komt nog. We hebben wel al een functie die alle classes zoekt die een interface implementeren: ModelBuilder.getSpringBeans().